bazel/cc_toolchains/clang_cc_toolchain_config.bzl

```
@@ -1101,13 +1100,15 @@ def cc_local_toolchain_suite(name, cpus):
     )
 
     # Create the individual local toolchains for each CPU.
-    for cpu in cpus:
+    for (os, cpu) in configs:
+        config_name = name + "_" + os + "_" + cpu
```

```suggestion
        config_name = "{0}_{1}_{2}".format(name, os, cpu)
```

Suggesting because there are so many values, mainly due to the above `local--` comment.